homeassistant/components/frontend/templates/index.html

```diff
@@ -78,6 +78,16 @@
     </div>
     <home-assistant icons='{{ icons }}'></home-assistant>
     {# <script src='/static/home-assistant-polymer/build/_demo_data_compiled.js'></script> #}
+    <script>
+      var compatibilityRequired = (
+        typeof Object.assign != 'function');
```

Comment:
This expression can be expanded as more polyfills are added. If any of them are necessary we can load the file.